app/views/spree/admin/orders/bulk_management.html.haml

```diff
@@ -177,7 +177,7 @@
           %td.email{ 'ng-show' => 'columns.email.visible' } {{ line_item.order.email }}
           %td.phone{ 'ng-show' => 'columns.phone.visible' } {{ line_item.order.phone }}
           %td.date{ 'ng-show' => 'columns.order_date.visible' } {{ line_item.order.completed_at }}
-          %td.producer{ 'ng-show' => 'columns.producer.visible' } {{ line_item.supplier.name }}
+          %td.producer{ 'ng-show' => 'columns.producer.visible', "ng-bind-html": 'line_item.supplier.name' }
```

Comment:
Doesn't this make us vulnerable to code injection? What if someone puts a `<script>` tag in their enterprise name?

Reply:
That's a good question @mkllnk. I'll need to do some research before I can reply to this. It might be particularly relevant as we're following this approach elsewhere too.

Reply:
I just tried to reproduce this bug but I couldn't. I wonder where the HTML code is coming from. Maybe it got copied in on the enterprise settings page?

Reply:
My bad, I've skipped creating an issue for this one - sorry for that.

But yes, that's it - it should come from the enterprise settings page. Visit `/admin/enterprises/<enterprise_name>/edit` and add some unusual characters to the enterprise name. Then, visit the bulk order management page. You should be able to see bad character rendering:

![image](https://github.com/user-attachments/assets/71ad07f1-0aa9-4e62-9ba9-b782c7f59039)

Reply:
> Doesn't this make us vulnerable to code injection?

I think we're good here - following the discussion security in a closed Slack channel.

> I just tried to reproduce this bug but I couldn't.

@mkllnk Did you manage to reproduce it in the meantime?

Reply:
No, I didn't. But if this is a fix in your environment then I'm happy with that. All good.